_kosmorro/main.py

```diff
@@ -38,6 +38,7 @@
     OutOfRangeDateError as DateRangeError,
 )
 from _kosmorro.i18n.utils import _, SHORT_DATE_FORMAT
+from .terms import terms as term_dictionary
```

Comment:
You can remove the alias, we don't need it here :)

```suggestion
from .terms import terms
```